src/nimodinst/system_tests/test_system_nimodinst.py

```diff
@@ -4,28 +4,29 @@
 import re
 
 
-def test_bad_device_family():
+def test_bad_device_family(family):
     with nimodinst.Session('FAKE') as session:
         assert len(session) == 0
 
 
-def test_no_device_family():
-    with nimodinst.Session('') as session:
+def test_no_device_family(family):
```

Comment:
This doesn't seem to do what the name says. It says no device family but then opens the session using the family